javascript/reactjs-todo-davinci/client/components/davinci-client/readonly.js

```
@@ -1,9 +1,47 @@
+ * Copyright (c) 2025 - 2026 Ping Identity Corporation. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
 import React from 'react';
+import { interpolateRichContent } from '../utilities/rich-content';
+
+export default function ReadOnly({ collector }) {
+  const collectorType = collector.type;
+  const output = collector.output;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)